api/types/github.go

```diff
@@ -275,11 +292,41 @@ func (c *GithubConnectorV3) MapClaims(claims GithubClaims) ([]string, []string,
 		for _, team := range teams {
 			// see if the user belongs to this team
 			if team == mapping.Team {
+				if !isEnterprise {
+					usesSSO, err := c.checkOrgSSO(mapping.Organization)
```

Comment:
Oops, yeah I'll move the check out of that loop.

I agree, I wanted to check enterprise status in this file but importing `github.com/gravitational/teleport/lib/modules` caused an import cycle.

That's a good idea, I agree that the check should be done when the connector is created. I do think there's value by also making this check when a user tries to log on, as the Github org could have enabled SSO since the connector was created.

Reply:
I agree that when the connector is created is not enough.

Reply:
Done